src-built-in/components/singleInputDialog/src/app.jsx

```diff
@@ -6,140 +6,8 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "../../../../assets/css/finsemble.css";
 
-import { FinsembleDialog, FinsembleDialogQuestion, FinsembleDialogTextInput, FinsembleDialogButton } from "@chartiq/finsemble-react-controls";
-/**
- * This is our standard input dialog. It gives the user a single button and a text field, and ferries that data back to the window that opened it.
- *
- * @class SingleInputDialog
- * @extends {React.Component}
- */
-class SingleInputDialog extends React.Component {
-	constructor() {
-		super();
-		this.bindCorrectContext();
-		this.state = {
-			inputLabel: "No question.",
-			affirmativeResponseLabel: "Yes",
-			/** Our input uses the HTML autoFocus attribute. This only fires when the element is rendered for the 1st time. Since we re-use the dialogs, the 2nd time the dialog is shown, autoFocus will not trigger. To force it to trigger, we change this boolean when a response is sent back to the opener, which unmounts the element from the dom. The next time the dialog is used, we will render the element again, and autoFocus will trigger. */
-			renderInput: false
-		};
-		document.body.addEventListener("keydown", this.handleKeydownOnBody);
-	}
+import { SingleInputDialog } from "@chartiq/finsemble-ui/lib/components/singleInputDialog/singleInputDialog";
```

Comment:
This can just be `import { SingleInputDialog } from "@chartiq/finsemble-ui/lib/components`

Reply:
Done!